ui/components/app/signature-request-siwe/signature-request-siwe.js

```diff
@@ -101,38 +103,25 @@ export default function SignatureRequestSIWE({
       ) : null}
       <Message data={formatMessageParams(parsedMessage, t)} />
       {!isMatchingAddress && (
-        <ActionableMessage
-          className="signature-request-siwe__actionable-message"
-          type="warning"
-          message={t('SIWEAddressInvalid', [
+        <BannerAlert severity={SEVERITIES.WARNING}>
+          {t('SIWEAddressInvalid', [
             parsedMessage.address,
             fromAccount.address,
           ])}
-          iconFillColor="var(--color-warning-default)"
-          useIcon
-          withRightButton
-          icon={<Icon name="danger" color={IconColor.warningDefault} />}
-        />
+        </BannerAlert>
       )}
       {!isSIWEDomainValid && (
-        <ActionableMessage
-          className="signature-request-siwe__actionable-message"
-          type="danger"
-          message={
-            <>
-              <p
-                className="typography--weight-bold"
-                style={{ display: 'inline' }}
-              >
-                {t('SIWEDomainInvalidTitle')}
-              </p>{' '}
-              {t('SIWEDomainInvalidText')}
-            </>
-          }
-          iconFillColor="var(--color-error-default)"
-          useIcon
-          icon={<Icon name="danger" color={IconColor.errorDefault} />}
-        />
+        <BannerAlert
+          severity={SEVERITIES.DANGER}
+          marginLeft={4}
+          marginRight={4}
+          marginBottom={4}
+        >
+          <Text variant={TextVariant.bodyMdBold}>
+            {t('SIWEDomainInvalidTitle')}
+          </Text>{' '}
+          <Text variant={TextVariant.bodyMd}>{t('SIWEDomainInvalidText')}</Text>
```

Comment:
nit: default variant for the `Text` component is `bodyMd` so shouldn't need variant here
```suggestion
          <Text>{t('SIWEDomainInvalidText')}</Text>
```

Reply:
updated [9b98162](https://github.com/MetaMask/metamask-extension/pull/18207/commits/9b98162d652667f299d24cca71f4ed8d0269d5c9)